scripts/debug_navigation_native.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This script replaces SPM checked out MapboxNavigationNative framework for simulator with a debug version.
+
+if [[ -z $1 || -z $2 ]]; then 
+    echo "Usage $0 <path to MapboxNavigationNative git repository> <simulator xcodebuild destination>"
+    exit 1
+fi
+
+set -x
+set -e
+
+# Install a tool which parses JSONs
+brew list jq || brew install jq
+
+NATIVE_PATH=$1
+DESTINATION=$2
+NATIVE_VERSION=$(jq '.object.pins[] | select(.package=="MapboxNavigationNative") | .state.version' ./Package.resolved | sed 's/^"\(.*\)"$/\1/')
```

Comment:
I guess you can inline a python code here and not install `jq`

Reply:
thank you for suggestion, implemented